src/style.css

```diff
@@ -1 +1,73 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #3D4E61;
+  --border-color: #E1E7ED;
+}
+
+.search-bar {
+  margin-top: 20px;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--border-color);
+  background: #FFF;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  border-radius: 4px;
+}
+
+.search-bar--small {
+  height: 42px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  transition: var(--border-color) 0.3s;
+  padding-left: 62px;
+  font-size: 16px;
+  color: var(--main-color);
+  background: url('images/Search.svg') no-repeat 26px center / 20px 20px;
+  position: absolute;
+}
+
+::placeholder {
```

Comment:
on tests, it's easy to see that font styles don't match the design:
![Screenshot from 2024-01-04 19-47-57](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/c43032ef-22c2-4612-ac14-a926524e2e7b)
![Screenshot from 2024-01-04 19-47-54](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/75c23db3-ac51-4b98-a0be-1464ab0e61f3)

try to fix it